hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -357,6 +363,58 @@
         }
       ]
     },
+    { name: "EXTHT_HI_THRESHOLDS",
+      desc: "External health test high thresholds register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen:   "REGEN",
+      fields: [
+        { bits: "15:0",
+          name: "FIPS_EXTHT_HI_THRESH",
+          desc: '''This is the threshold size for the external health test.
+                   This is value used in normal mode when entropy is being tested in
+                   FIPS/CC compliance mode.
+                   The default value is 0.
+                '''
+          resval: "0x0"
+        }
+        { bits: "31:16",
+          name: "BYPASS_EXTHT_HI_THRESH",
+          desc: '''This is the threshold size for the external health test
+                   running is bypass mode. This mode is active after reset for the
+                   first and only test run, or when this mode is programmed by firmware.
+                   The default value is .375 of the FIPS threshold.
```

Comment:
this appears to represent a fractional number. would it make sense to say something about what the number format for the thresholds is? is it `UQ0.16`?

Reply:
There is no format there, it is open for external use. Setting default value to zero.